README.md

```diff
@@ -74,7 +74,7 @@ $ kubectl kustomize config/default | kubectl apply -f -
 
 # License
 
-The Blixt control-plane components are licensed under [Apache License, Version
+The Blixt controlplane components are licensed under [Apache License, Version
```

Comment:
This didn't need to change, it's ok to refer to the control plane as `control-plane` even if the image name is `controlplane`.